### PR TITLE
Workaround bug with issue search that ignores `repo` params when `org` specified

### PR DESCRIPTION
**Pull request by nholden**
👋  Hi! Last week, we noticed that our first responder bot was commenting on repos outside of the one that we had specified with the `include-repos` option. As it turns out, there's a bug with issue search that causes the `repo` param to be ignored when an `org` is specified.

To illustrate the issue search bug, [a search for `repo:rails/sprockets`](https://github.com/issues?q=repo%3Arails%2Fsprockets) returns only issues from the `rails/sprockets` repo. [A search for `repo:rails/sprockets org:rails`](https://github.com/issues?q=repo%3Arails%2Fsprockets+org%3Arails) ignores the `repo` param and returns issues from across the `rails` org.

This PR works around that bug by only including an `org` in the first responder search query when there is no `include-repos` specified.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -95,11 +95,17 @@ async function getTeamPingIssues (octokit, org, team, authors, commenters, since
     includeRepos.forEach(elem => {
       query = query.concat(`+repo%3A${elem}`)
     })
-  } else if (ignoreRepos.length > 0) {
+  } else {
+    // Limit repos to those in org
+    query.concat(`+org%3A${org}`)
```
Comment (rachmari) [inline index.js]:
```suggestion
    query = query.concat(`+org%3A${org}`)
```

**Review by rachmari — COMMENTED** [review]
@nholden thanks for fixing this! Thinking about this a bit more, I think the fact that a query that includes `repo:<org/owner><repo-name>` and `org:<org-name>` prefers the org over the repo is actually intentional. The `repo` and `org` qualifier specify which repos to search within so if both org and repo exist it's choosing the larger group of the two. 

After `@itsbagpack` added the new `include-repos` input parameter I didn't realize that we would need to remove `org`. 

There was one tiny suggestion that we'll need to fix before merging this. Also, can you run `npm run build` in your local checkout to generate the `dist/` files? If you don't have this cloned locally, I can do that right after I merge this PR. I can then tag the new release for your team to use.

File: index.js (hunk 1 above)
Comment (nholden) [reply index.js]: Whoops, great catch! 😅

File: dist/index.js [hunk 2]
```diff
@@ -2187,7 +2193,7 @@ module.exports = require("https");
...
...js --devtool source-map","build:ts":"npm run -s update-endpoints:typescript","coverage":"nyc report --reporter=html && open coverage/index.html","generate-bundle-report":"webpack-bundle-analyzer dist/bundle-stats.json --mode=static --no-open --report dist/bundle-report.html","lint":"prettier --check '{lib,plugins,scripts,test}/**/*.{js,json,ts}' 'docs/*.{js,json}' 'docs/src/**/*' index.js README.md package.json","lint:fix":"prettier --write '{lib,plugins,scripts,test}/**/*.{js,json,ts}' 'docs/*.{js,json}' 'docs/src/**/*' index.js README.md package.json","postvalidate:ts":"tsc --noEmit --target es6 test/typescript-validate.ts","prebuild:browser":"mkdirp dist/","pretest":"npm run -s lint","prevalidate:ts":"npm run -s build:ts","start-fixtures-server":"octokit-fixtures-server","test":"nyc mocha test/mocha-node-setup.js \"test/*/**/*-test.js\"","test:browser":"cypress run --browser chrome","update-endpoints":"npm-run-all update-endpoints:*","update-endpoints:fetch-json":"node scripts/update-endpoints/fetch-json","update-endpoints:typescript":"node scripts/update-endpoints/typescript","validate:ts":"tsc --target es6 --noImplicitAny index.d.ts"},"types":"index.d.ts","version":"16.43.1"};
```
Comment (nholden) [inline dist/index.js:2196]: I don't think we want this change, but it was generated by `npm run build`. Any suggestions?

**Comment by nholden** [comment]
Thanks for the feedback, @rachmari!

> Thinking about this a bit more, I think the fact that a query that includes repo:<org/owner><repo-name> and org:<org-name> prefers the org over the repo is actually intentional. The repo and org qualifier specify which repos to search within so if both org and repo exist it's choosing the larger group of the two.

This is surprising to me. Most of the time, adding [a qualifier](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests) makes the query more specific and will return fewer results, but in this case, adding `org:` makes the query more generic and returns more results. Guess that's not really a concern of this PR though, so we can chat with the Issues team about that.

> Also, can you run npm run build in your local checkout to generate the dist/ files? If you don't have this cloned locally, I can do that right after I merge this PR.

Just did this! As I mentioned in a comment, it looks like it generated a change that we may not want to include. Feel free to modify as you see fit.

**Comment by nholden** [comment]
Doing some old PR cleanup. Closing this one out! 😁

**Pull request closed without merging**